README.md

```diff
@@ -1,24 +1,143 @@
-# README
+<a name="readme-top"></a>
+<!-- TABLE OF CONTENTS -->
+<div align="center">
+  
+</div>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
 
-Things you may want to cover:
+# 📗 Table of Contents
 
-* Ruby version
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
 
-* System dependencies
+- [📝 License](#license)
 
-* Configuration
 
-* Database creation
+# 📖 Budget App <a name="about-project"></a>
 
-* Database initialization
+**Budget App** is a Ruby on Rails capstone project which you can create new categories, you can write new transactions and deploy it the categories and finally you can control how much money you spend each category.
 
-* How to run the test suite
+## Video Link 🚀
 
-* Services (job queues, cache servers, search engines, etc.)
 
-* Deployment instructions
+## Render Deploy 🚀
 
```

Comment:
- [ ] Please deploy the latest working version of your app and attach the link to the README file as per project requirements. You can use `onrender` to easily deploy a RoR app.

Reply:
well resolved! ✔️